### DISCUSSION TITLE
Version 2.981

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1058] [General]
- Add Sponsorblock support by James Cullum
- Update NewPipeExtractor to fix recent Youtube changes
- Updated translations for:
  - Albanian - Besnik Bleta
  - Arabic - Khalid Qasim
  - Basque - Julio Montero
  - Croatian - Milo Ivir
  - Czech - Jáchym Řezáč
  - Dutch - Jon Bedet
  - French - J. Lavoie
  - German - 3ole
  - Hindi - Tan
  - Hungarian - Szia Tomi
  - Italian - Tommaso Giutna
  - Norwegian - Allan Nordhøy
  - Polish - Michal L
  - Romanian - Igor Sorocean
  - Russian - lk2322
  - Simplified Chinese - Eric
  - Sinhala - Kaveesha
  - Tamil - Bharath
  - Thai - Sim WC
  - Traditional Chinese - Jeff Huang and zica
  - Turkish - Oğuz Ersen
  - Ukrainian - Ihor Hordiichuk

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/v2.981'>Version 2.981</a>.</em>

**Comment by andercard0** [comment]
Getting erros:
- Program has stopped to work> to reproduce: Try to import my youtube's subscriptions
- Search does not work at all> to reproduce: try to search something nothing happens

Maybe this isn't the correct place to post this but whatever! 
Too lazy to open a issue :p
